profile/2018-06-01-profile.json

```diff
@@ -0,0 +1,655 @@
+                             "locations/longTermRetentionBackups",
+                             "locations/longTermRetentionPolicyOperationResults",
+                             "locations/longTermRetentionPolicyAzureAsyncOperation",
+                             "locations/longTermRetentionBackupOperationResults",
+                             "locations/longTermRetentionBackupAzureAsyncOperation"
+                         ],
+
+                         "2017-10-01-preview": [
+                            "locations/capabilities",
+                            "locations/databaseAzureAsyncOperation",
+                            "locations/databaseOperationResults",
+                            "servers/databases",
+                            "locations/elasticPoolAzureAsyncOperation",
+                            "locations/elasticPoolOperationResults",
+                            "servers/elasticpools",
+                            "servers/databases/vulnerabilityAssessments"
+                         ]
+
+            },
+
+            "microsoft.storage": {
+
+                         "2017-10-01": [
+                             "checkNameAvailability",
+                             "operations",
+                             "storageAccounts",
+                             "storageAccounts/blobServices",
+                             "storageAccounts/fileServices",
+                             "storageAccounts/listAccountSas",
+                             "storageAccounts/listServiceSas",
+                             "storageAccounts/queueServices",
+                             "storageAccounts/tableServices",
+                             "usages"
+                         ]
+            },
+
+            "microsoft.web": {
+
+                          "2016-03-01": [
+                              "availableStacks",
+                              "certificates",
+                              "checkNameAvailability",
+                              "deploymentLocations",
+                              "georegions",
+                              "ishostingenvironmentnameavailable",
+                              "ishostnameavailable",
+                              "isusernameavailable",
+                              "listSitesAssignedToHostName",
+                              "operations",
+                              "publishingUsers",
+                              "runtimes",
+                              "sites/premieraddons",
+                              "sourceControls"
+
+                          ],
+
+                          "2016-08-01": [
+                            "sites",
+                            "sites/extensions",
+                            "sites/hostNameBindings",
+                            "sites/instances",
+                            "sites/instances/extensions",
+                            "sites/slots",
+                            "sites/slots/extensions",
```

Comment:
Seems to be sites/slots/instances/extensions

Reply:
Have that one below as well, both of these exist according to my list

Reply:
Based on public Azure provider list, it doesn't exist.

Reply:
The Azure provider list provided to me by Josh has both. I can reach out to service teams and see, but if there is no swagger I can delete, just let me know.

Reply:
FYI @joshgav